tck/src/main/java/org/eclipse/microprofile/reactive/streams/operators/tck/arquillian/ReactiveStreamsCdiTck.java

```diff
@@ -23,8 +23,8 @@
 import org.eclipse.microprofile.reactive.streams.operators.spi.ReactiveStreamsEngine;
 import org.reactivestreams.tck.TestEnvironment;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Inject;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
```

Comment:
> @danielkec I thought the tcks does not depend on Jakarta specs. Can you confirm?

@Emily-Jiang I can't, tcks does depend on jakarta apis

Reply:
@danielkec thanks, I missed this file. In this case, the spec needs to do major version change. Sorry about this. Can you push back the major version bump back? Once you are done, I can merge your PR and do a  RC1 to check out the artifacts. Thank you for doing this!

Reply:
I have merged your PR there. I will do a release soon for microprofile-parent and then push to maven central.